### PR TITLE
added custom docker environment setting

### PR DESCRIPTION
**Pull request by yvonnebarthp**

### CODE REVIEW

File: docs/how-to/AdjustingDockerfile.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Customizing the base docker image (mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210922.v1)
```
Comment (mariamedp) [inline docs/how-to/AdjustingDockerfile.md:1]: Can we specify somehow that we are talking about environments to be used in training/inference pipelines?
Customizing a docker image is something that could also apply to webservices deployment, but that would be done differently, not via `configuration/environments/` (issue #29 is tracking this enhancement).

**Review by mariamedp — CHANGES_REQUESTED** [review]
Thanks! just a few small details

**Review by yvonnebarthp — COMMENTED** [review]
Updated naming conventions, versions, etc.

File: docs/how-to/FAQ.md [hunk 2]
```diff
@@ -4,4 +4,9 @@
...
 If you see the error below. You have to ensure that the service connection is created at the Azure Machine Learning Workspace level and not Subscription level
 
-![faq-error-invoke](../media/FAQ-invoke.error.png)
\ No newline at end of file
+![faq-error-invoke](../media/FAQ-invoke.error.png)
+
+## I would like to adjust the Docker Image. How can I change the Dockerfile?
+
+Documentation can be found here: 
+**[Custom Dockerfile](dstoolkit-mlops-base/docs/how-to/AdjustingDockerfile.md)**
```
Comment (mariamedp) [inline docs/how-to/FAQ.md:12]: The link doesn't work (check [here](https://github.com/microsoft/dstoolkit-mlops-base/blob/4d450dcce572c6f62ade8097826e33a892b29e8d/docs/how-to/FAQ.md)).

**Review by mariamedp — APPROVED** [review]

**Review by mariamedp — APPROVED** [review]

**Pull request merged**